src/components/common/ColonyMembers/MembersFilter/MembersFilter.tsx

```diff
@@ -46,18 +45,19 @@ const MembersFilter = ({ handleFiltersCallback, isRoot }: Props) => {
     <>
       <hr className={styles.divider} />
       <Form
-        initialValues={{
+        defaultValues={{
           memberType: MemberType.All,
           verificationType: VerificationType.All,
         }}
         onSubmit={() => {}}
-        enableReinitialize
       >
-        {({ resetForm, values }: FormikProps<FormValues>) => {
+        {({ formState: { isSubmitting, isValid }, getValues, reset }) => {
```

Comment:
Not sure it makes sense to expose an `isSubmitting` prop here given form submission does nothing. Also there's no validation schema so the form will never be invalid.